modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -182,11 +183,12 @@ export interface SignTransactionOptions extends BaseSignTransactionOptions {
       }[];
     };
   };
+  // xprv
   prv: string;
-  userKeychain?: Keychain;
-  backupKeychain?: Keychain;
-  bitgoKeychain?: Keychain;
-  taprootRedeemIndex?: number;
+  // xpubs (user, backup, bitgo)
+  pubs: Triple<string>;
```

Comment:
Should this be called `xpubs` to clearly identify it for what it is? Without context I'd expect these to be regular pubkeys. I guess on the other hand we have `prv` which refers to an xprv, but I wonder if we should just rename that to `xprv` as well and eliminate any possibility for confusion.

Reply:
The problem is that `pub` is part of the generic interface, so I can't just rename it.

Reply:
Hm OK. In that case it might be nice to make these comments use the `/** */` style, that way many IDEs will associate that comment with the property so if anyone ever mouses over it they'll get a tooltip with this comment. I know that kind of thing is helpful for me.